src/grid/submission/SubmissionGridHeader.component.ts

```diff
@@ -31,21 +32,21 @@ export class SubmissionGridHeaderComponent extends GridHeaderComponent {
   }
 
   // Set header for both component and column
-  setHeader(column?: any, component?: ExtendedComponentSchema, sort?: EventEmitter<any>) {
-    const key = column ? column.path : `data.${component.key}`;
+  setHeader(column?: GridColumn, component?: ExtendedComponentSchema, sort?: any) {
+    const path = column ? column.path : `data.${component.key}`;
     const label = column ? column.label : component.label;
 
     this.headers.push({
       label: label,
-      key: key,
+      key: path,
       sort: sort || '',
```

Comment:
@vlad-shusterman shouldn't just `sort: sort` be enough here? If it's undefined, `header.sort` will then also be undefined, which is acceptable, isn't it?